packages/shared-state-wifi_links_info/files/usr/bin/shared-state-publish_wifi_links_info

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/lua
+
+--! LibreMesh
+--! Copyright (C) 2019  Gioacchino Mazzurco <gio@altermundi.net>
+--! Copyright (C) 2019  Javier Jorge <jjorge@inti.gob.ar>
+--!
+--! This program is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU Affero General Public License as
+--! published by the Free Software Foundation, either version 3 of the
+--! License, or (at your option) any later version.
+--!
+--! This program is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU Affero General Public License for more details.
+--!
+--! You should have received a copy of the GNU Affero General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+local JSON = require("luci.jsonc")
+local node_status = require ("lime.node_status")
+local network = require ("lime.network")
+
+function get_wifi_links_info()
+	local stations = node_status.get_stations()
+	local links = {}
+	for _, station in ipairs(stations) do
+		macparts = network.get_mac(station.iface)
+		src_macaddr = table.concat(macparts,":")
+		local station_stats = node_status.get_station_stats(station)
+		table.insert(links, {src_mac=src_macaddr ,dst_mac=station.station_mac,signal=station_stats.signal,chains=station_stats.chains,rx_rate=station_stats.rx_rate,tx_rate=station_stats.tx_rate } )
```

Comment:
Can you split this like so it fits in 80 column or less? Please split it in a way that make it easier to read ;)